### ISSUE TITLE
Not compatible with node 18

### ISSUE THREAD

**Issue by randomnessdev** [jclusso/strapi-plugin-netlify-deployments#2]
Hello,

I got this error:

```
error strapi-plugin-netlify-deployments@2.0.0: The engine "node" is incompatible with this module. 
Expected version ">=12.x.x <=16.x.x". Got "18.14.0"
```

Is it possible to make it compatible with node 18 ?

**Comment by FrancoRueta** [comment]
i got the same problem and had to uninstall this plugin without even being able to try it out. it would've been great to be able to deploy to netlify, hope to come by this project and see it being upgraded to test it out.

**Comment by maszatkavics** [comment]
Same issue here, please give us a solution for that! Thanks.

**Comment by jclusso** [comment]
Sorry guys, I'm not actively using this. I'll see if it's a quick fix when I have some time or if someone wants to open a PR that's welcome too!

**Comment by robclancy** [comment]
@jclusso it has a pr above ^ (I'm gonna fork anyway)

**Comment by jclusso** [comment]
Resolved in #6.
